src/uct/ib/rdmacm/rdmacm_cm.c

```diff
@@ -437,7 +437,7 @@ static void uct_rdmacm_cm_handle_error_event(struct rdma_cm_event *event)
         log_level = UCS_LOG_LEVEL_DEBUG;
     } else {
         status    = UCS_ERR_IO_ERROR;
-        log_level = UCS_LOG_LEVEL_ERROR;
+        log_level = uct_rdmacm_cm_ep_get_cm(cep)->super.config.failure_level;
```

Comment:
pls set status like in https://github.com/openucx/ucx/blob/9aead1c9d79d2d9eabceac0647033f8851ac3198/src/uct/ib/rdmacm/rdmacm_cm.c#L654, instead of IO_ERROR

Reply:
> pls set status like in https://github.com/openucx/ucx/blob/9aead1c9d79d2d9eabceac0647033f8851ac3198/src/uct/ib/rdmacm/rdmacm_cm.c#L654, instead of IO_ERROR

backported from master